tests/datamock/mock_tradeapi.py

```diff
@@ -19,13 +20,19 @@ def list_assets(tradeapi):
 
 
 def get_barset(tradeapi):
```

Comment:
Maybe we can rename this method as well to `get_bars`?

Reply:
good call

Reply:
actually, it already is named get_bars, is GH having a weird issue? :thinking:

Reply:
That's weird. I see it on my side now.